### PR TITLE
Remove unnecessary comments on issue templates

### PR DESCRIPTION
**Pull request by Joehuu**
The labels should be self-explanatory already.

The regular issue template should've been removed in the past (when the other templates were made to separate it).

---

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/missing-for-live-issues.md [hunk 1]
```diff
@@ -2,9 +2,6 @@
...
 about: Let us know the features you need which are available in osu-stable but not lazer
 ---
+**Describe the missing feature:** 
 
-<!-- After you fill in all information, delete all comments in the issue -->
-
-**Describe the feature:** <!-- Describe the missing game feature -->
-
-**Designs:** <!-- Attach screenshots of how the feature is supposed to look like. For illustrative purpose only; final designs are usually re-imagined from scratch. -->
+**Designs:**
```
Comment (Aergwyn) [inline .github/ISSUE_TEMPLATE/missing-for-live-issues.md]:
```suggestion
**Proposal designs of the feature:** 
```

File: .github/ISSUE_TEMPLATE/bug-issues.md [hunk 2]
```diff
@@ -2,13 +2,10 @@
 name: Bug Report
 about: For issues regarding encountered game bugs
```
Comment (Aergwyn) [inline .github/ISSUE_TEMPLATE/bug-issues.md]:
```suggestion
about: Issues regarding encountered bugs.
```

File: .github/ISSUE_TEMPLATE/crash-issues.md [hunk 3]
```diff
@@ -2,15 +2,12 @@
 name: Crash Report
 about: For issues regarding game crashes or permanent freezes
```
Comment (Aergwyn) [inline .github/ISSUE_TEMPLATE/crash-issues.md]:
```suggestion
about: Issues regarding crashes or permanent freezes.
```

File: .github/ISSUE_TEMPLATE/missing-for-live-issues.md [hunk 4]
```diff
@@ -2,9 +2,6 @@
 name: Missing for Live
 about: Let us know the features you need which are available in osu-stable but not lazer
```
Comment (Aergwyn) [inline .github/ISSUE_TEMPLATE/missing-for-live-issues.md]:
```suggestion
about: Features which are available in osu!stable but not yet in osu!lazer.
```

File: .github/ISSUE_TEMPLATE/feature-request-issues.md [hunk 5]
```diff
@@ -2,9 +2,6 @@
 name: Feature Request
 about: Let us know what you would like to see in the game!
```
Comment (Aergwyn) [inline .github/ISSUE_TEMPLATE/feature-request-issues.md]:
```suggestion
about: Features you would like to see in the game!
```

**Review by Aergwyn — COMMENTED** [review]
It's less noisy now. I like that.

**Comment by peppy** [comment]
I was kinda hoping to remove them altogether, but let's see how this goes.

**Pull request merged**